lib/govuk_publishing_components/presenters/contextual_navigation.rb

```diff
@@ -8,7 +8,7 @@ class ContextualNavigation
       # @param request_path `request.path`
       def initialize(content_item, request_path)
         @content_item = content_item
-        @request_path = request_path
+        @request_path = content_item['document_type'] === "simple_smart_answer" ? content_item['base_path'] : request_path
```

Comment:
Nitpick: You could break out the simple smart answer check to make this line a bit shorter.

```
@request_path = simple_smart_answer? ? content_item['base_path'] : request_path

def simple_smart_answer?
  content_item['document_type'] === "simple_smart_answer" 
end
```